exercises/map/map.js

```diff
@@ -24,17 +24,26 @@ function map(collection, fn) {
   let results = [];
 
   for (let item of collection) {
-    // This is your job. :)
+    results.push(fn(item));
```

Comment:
👍 

From now on, you should feel free to use https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map